stories/Accordion.stories.mdx

```diff
@@ -32,10 +32,19 @@ export const Template = (args) => html`
             <button class="jazz-accordion__button" aria-controls="content4">Vivamus magna justo, lacinia eget consectetur sed, convallis at tellus. Curabitur aliquet quam id dui posuere blandit.</button>
           </h2>
           <div class="jazz-accordion__content" id="content4">
-              Nulla porttitor accumsan tincidunt. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices 
-              posuere cubilia Curae; Donec velit neque, auctor sit amet aliquam vel, ullamcorper sit amet ligula. 
+              Nulla porttitor accumsan tincidunt. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices
+              posuere cubilia Curae; Donec velit neque, auctor sit amet aliquam vel, ullamcorper sit amet ligula.
               Curabitur arcu erat, accumsan id imperdiet et, porttitor at sem.
           </div>
+          <h2>
```

Comment:
Can you add an example of this in the documentation site example1 code so we can easily test this from there as well?